classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
```

Comment:
- [ ] Please provide `attr_reader :students` when you use attr_reader, it only creates a getter method So You can use students to retrieve the value of the @Students instance variable.